lib/api/src/rest/schema.rs

```diff
@@ -123,4 +128,5 @@ pub enum NamedVectorStruct {
     Default(segment::data_types::vectors::DenseVector),
     Dense(segment::data_types::vectors::NamedVector),
     Sparse(segment::data_types::vectors::NamedSparseVector),
+    // No support for multi-dense vectors in search
```

Comment:
What's about advanced search API like Discovery? They have another endpoint and I don't understand why we won't support multivector for discovery/recommend/etc do untrivial enum here with mismatch

Reply:
Got it, mark as resolved